client/controllers/WebPublisherContentListsController.js

```diff
@@ -42,6 +42,21 @@ export function WebPublisherContentListsController($scope, publisher, modal, $ti
             });
         }
 
+        /**
+         * @ngdoc method
+         * @name WebPublisherContentListsController#getThumbnail
+         * @param {Object} article - article content object
+         * @description Sets the active view name to the given value
+         */
+        getThumbnail(article) {
+            let base = article.tenant.subdomain ? 'http://' + article.tenant.subdomain + '.' + article.tenant.domainName : 'http://' + article.tenant.domainName;
```

Comment:
it should be without the protocol: `://`.  The protocol can be omitted to not throw any warnings  I guess.

Reply:
true